Sources/GetStartedApp/DatabaseManager.swift

```diff
@@ -19,7 +19,6 @@
 // operations is not needed by calling functions (i.e. the Controller class)
 // for different vendors.
 protocol DatabaseManager {
-  func getVisitors() -> [String]?
-  func addVisitors(user: [String: String]) -> Bool
+    func getVisitors() -> [String]?
+    func addVisitors(user: [String: String]) -> Bool
```

Comment:
This protocol could replace the `[String: String]` with a User with the changes followed through to any classes that use this protocol.